libs/email-service/src/main/java/uk/gov/di/ipv/core/library/service/EmailService.java

```diff
@@ -46,14 +55,20 @@ public void sendUserTriggeredIdentityResetConfirmation(
                         "Attempting to send user triggered identity reset confirmation email"));
         // This template ID can vary between production and the other environments, so it can't be
         // hardcoded
```

Comment:
If you do leave things like they are now, then this comment doesn't make sense any more as the template IDs are being hardcoded in the other methods.